cluster/fpindexcluster/manager.py

```diff
@@ -287,6 +291,62 @@ async def _apply_create_index(self) -> None:
             logger.exception("Error creating index %s", self.index_name)
             raise
 
+    async def _apply_update_operation(self, operation: UpdateOperation) -> None:
+        """Apply an UpdateOperation to fpindex by forwarding the HTTP request."""
+        logger.info(f"Applying UpdateOperation with {len(operation.changes)} changes for '{self.index_name}'")
+
+        # Convert our internal Change format to fpindex format
```

Comment:
If you use the `omit_defaults` option, you don't actually need to reformat changes.